core/platform/monitoring.go

```diff
@@ -0,0 +1,15 @@
+package platform
+
+// Observability keys
+const (
+	CapabilityIDKey        = "capabilityID"
+	TriggerIDKey           = "triggerID"
+	WorkflowIDKey          = "workflowID"
+	WorkflowExecutionIDKey = "workflowExecutionID"
+	WorkflowNameKey        = "workflowName"
+	WorkflowOwnerKey       = "workflowOwner"
+	StepIDKey              = "stepID"
+	StepRefKey             = "stepRef"
+)
+
+var OrderedLabelKeys = []string{StepRefKey, StepIDKey, TriggerIDKey, CapabilityIDKey, WorkflowExecutionIDKey, WorkflowIDKey}
```

Comment:
Maybe consider `Keys` as a prefix here too.

Also, the new iterator API gives us an opportunity to offer the same kind of iteraction without using a mutable variable. For example:
```suggestion
func KeysSorted() iter.Seq[string] {
	return return slices.Values([]string{StepRefKey, StepIDKey, TriggerIDKey, CapabilityIDKey, WorkflowExecutionIDKey, WorkflowIDKey})
}
```
Notice that using `Values` instead of `All` means the callers can drop the `_, ` too:
```go
for label := range platform.KeysSorted()
```